src/Sylius/Component/Order/Model/Adjustment.php

```diff
@@ -49,6 +49,9 @@ class Adjustment implements AdjustmentInterface
     /** @var string|null */
     protected $originCode;
 
+    /** @var array */
```

Comment:
I'm wondering if this value cannot be empty due to migration chages

Reply:
As I wrote in the description, there should be probably prepared a proper migration to fill this array for existing adjustments, so it should resolve this problem.